backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/repos/RegionsRepository.kt

```diff
@@ -1,20 +1,29 @@
 package app.ehrenamtskarte.backend.regions.database.repos
 
 import app.ehrenamtskarte.backend.common.database.sortByKeys
+import app.ehrenamtskarte.backend.projects.database.Projects
 import app.ehrenamtskarte.backend.regions.database.RegionEntity
 import app.ehrenamtskarte.backend.regions.database.Regions
+import org.jetbrains.exposed.sql.and
+import org.jetbrains.exposed.sql.select
 
 object RegionsRepository {
 
-    fun findAll() = RegionEntity.all()
+    fun findAllInProject(project: String): List<RegionEntity> {
+        val query = (Projects innerJoin Regions)
+            .slice(Regions.columns)
+            .select { Projects.project eq project }
+        return RegionEntity.wrapRows(query).toList()
+    }
+
+    fun findByIdsInProject(project: String, ids: List<Int>): List<RegionEntity?> {
+        val query = (Projects innerJoin Regions)
+            .slice(Regions.columns)
+            .select { Projects.project eq project and (Regions.id inList ids) }
+        return RegionEntity.wrapRows(query).sortByKeys({ it.id.value }, ids)
+    }
 
     fun findByIds(ids: List<Int>) =
         RegionEntity.find { Regions.id inList ids }.sortByKeys({ it.id.value }, ids)
 
-    fun findById(id: Int) =
-        RegionEntity.find { Regions.id eq id }.singleOrNull()
-
-    fun findByRegionIdentifiers(ids: List<String>) =
-        RegionEntity.find { Regions.regionIdentifier inList ids }.sortByKeys({ it.regionIdentifier }, ids)
-
```

Comment:
I think this endpoint should stay. Or was it unused?

Reply:
Both functions were unused (and they are no endpoints), so I think it is fine to remove them.